packages/core/src/components/way-input/way-input.scss

```diff
@@ -0,0 +1,92 @@
+:host {
+  /**
+    * @prop --border-radius: Border radius of the input
+    * @prop --border-width: Border width of the button
+    * @prop --border-style: Border style of the button
+    * @prop --border-color: Border color of the button
+    */
+  display: inline-block;
+
+  width: auto;
+
+  font-family: var(--way-font-family, inherit);
+  font-size: var(--way-font-size);
+  line-height: 1.25rem;
+
+  user-select: none;
+  vertical-align: top;
+  vertical-align: -webkit-baseline-middle;
+  pointer-events: auto;
+
+  font-kerning: none;
+
+  --border-radius: 5px;
```

Comment:
**issue**: Deze aub linken aan `var(--way-form-element-border-radius-medium);` zodat het consistent is met alle form elements.

Reply:
mmh ok, wilde geen nieuwe aanmaken, maar had toch liever net die pixel meer :) maar prima

Reply:
Mhh ja ik snap. Ik heb het op 4px vanwege dat dit ook in Compass design wordt gebruikt. Maar van mij mag onderliggende `--way-border-radius-medium` ook wel ingesteld worden op een rem die overeenkomt met 5px; (`0.313rem`)

Reply:
Had het aangepast, weet niet of je het al gezien had?